turbo.json

```diff
@@ -7,13 +7,8 @@
       "outputs": ["dist/**", ".next/**"]
     },
     "@skillrecordings/config#build": {
-      "dependsOn": [
-        "^build",
-        "generate",
-        "$CONVERTKIT_API_SECRET",
-        "$NEXT_PUBLIC_CONVERTKIT_TOKEN",
-        "$NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM"
-      ],
+      "dependsOn": ["^build", "generate"],
+      "env": ["CONVERTKIT_API_SECRET"],
```

Comment:
As of turbo 1.4+, env var dependencies can be listed in the "env" key
rather than in the "dependsOn" key.

Docs: https://turborepo.org/docs/reference/configuration#env